dapps/marketplace/src/pages/listings/Listings.js

```diff
@@ -294,6 +323,19 @@ export default withGrowthRewards(
 )
 
 require('react-styl')(`
+  .listings-menu-bar 
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+    padding: 0 1rem;
+    position: relative;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
I think the internal preference is for no semicolons these days. Not sure if this is covered by the linter 😐

Reply:
Cleaned this up.